static/allinone.css

```diff
@@ -246,3 +248,9 @@ margin-right: auto;
 font-size: 12px;
 list-style: none;
 }
+/* for mobile */
+@media screen and (max-width: 430px) {
```

Comment:
This change doesnt change anything 
![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/eac6f441-4297-4451-a906-f341f12123b1)


![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/3d82cd8b-85d9-4d1a-8276-fceade374e46)